pkg/hsskey/hsskey.go

```diff
@@ -182,6 +194,38 @@ func GetAllHss(warnings io.Writer, verboseDangerous bool, eepromPattern string,
 	return nil, fmt.Errorf("failed all HSS retrieval attempts")
 }
 
+// evaluateHssPath evaluates a filepath to return the file or contents if a directory.
+// This function evaluates the base pointer for symlinks.
+func evaluateHssPath(hssFiles string) ([]string, error) {
+	// Follow symlink if needed.
+	hssFiles, err := filepath.EvalSymlinks(hssFiles)
+	if err != nil {
+		return nil, fmt.Errorf("could not eval symlink %s", hssFiles)
+	}
+
+	hssFileArr := []string{}
+
+	fi, err := os.Stat(hssFiles)
+	if err != nil {
+		return nil, fmt.Errorf("failed to stat HSS files: %v", err)
```

Comment:
%w since it is a package?

Reply:
Oh, hadn't seen %w. Switched to it for cases in this file. Thanks.